lib/refile/backend/file_system.rb

```diff
@@ -34,7 +34,7 @@ def initialize(directory, max_size: nil, hasher: Refile::RandomHasher.new)
       # @return [Refile::File]      The uploaded file
       verify_uploadable def upload(uploadable)
         id = @hasher.hash(uploadable)
-        IO.copy_stream(uploadable, path(id))
+        ::File.open(path(id), "wb") { |dest| IO.copy_stream(uploadable, dest) }
```

Comment:
I had to use {} block here, because jruby was raising: `syntax error, unexpected kDO_BLOCK` with multiline do..end block, and rubocop disallowed multiline {} blocks.